flopy/modflow/mfupw.py

```diff
@@ -276,7 +277,7 @@ def write_file(self, check=True, f=None):
             if self.chani[k] < 1:
                 f_upw.write(self.hani[k].get_file_entry())
             f_upw.write(self.vka[k].get_file_entry())
-            if transient == True:
+            if transient is True:

```

Comment:
I'm sorry to beat you up on this, but I think the correct syntax is:

`if transient:`

Reply:
And then I think it's ready to go.  Thanks so much.

Reply:
No worries. codacy recommend either `if transient:` or `if transient is True:`. I had gone with the latter for readability.